Lib/fontTools/varLib/interpolatable.py

```diff
@@ -365,7 +393,21 @@ def main(args=None):
         help="Output report in JSON format",
     )
     parser.add_argument(
-        "inputs", metavar="FILE", type=str, nargs="+", help="Input TTF/UFO files"
+        "--quiet",
+        action="store_true",
+        help="Only exit with code 1 or 0, no output",
+    )
+    parser.add_argument(
+        "--ignore-missing",
+        action="store_true",
```

Comment:
I wonder if maybe this should be True by default, i.e. expect inputs to be sparse by default, and maybe we add an option to `--report-missing-glyphs`.

Reply:
I would argue it is an improvement of the tool, more than a breaking change ;)
@behdad WDYT?

Reply:
I'm fine either way.

Reply:
I'm fine either way as well, but speaking as someone who is using this script as a sort of pre-build check, I would personally rather opt-in to a behavior than to opt-out of it, especially since in this case it's changing the default behavior. 

I was also worried about causing trouble for other scripts that used this script in some way — for example fontbakery uses it for a check.

Reply:
I'm fine to keep it opt-in

Reply:
Ok, seems like we're agreed — will resolve this conversation